py/k8s/dev/azure/r2r.yaml

```diff
@@ -0,0 +1,140 @@
+# Config Map, Secret, Deployment and Service for R2R
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: r2r-secret
+  namespace: r2r-dev
+  labels:
+    app: r2r
+data:
+  R2R_POSTGRES_USER: cG9zdGdyZXM=
+  R2R_POSTGRES_PASSWORD: cG9zdGdyZXM=
+#  R2R_POSTGRES_USER: YW1wbGlmaWRiX2FkbWlu
+#  R2R_POSTGRES_PASSWORD: QW1wbGlmaVBhc3N3b3JkODU2Mzc5
+
+  HATCHET_CLIENT_TOKEN: ZXlKaGJHY2lPaUpGVXpJMU5pSXNJQ0pyYVdRaU9pSjBibE5yWm5jaWZRLmV5SmhkV1FpT2lKb2RIUndPaTh2YUdGMFkyaGxkQzF6ZEdGamF5MWhjR2s2T0RBNE1DSXNJQ0psZUhBaU9qUTRPRFV3TmpneE5qVXNJQ0puY25CalgySnliMkZrWTJGemRGOWhaR1J5WlhOeklqb2lhR0YwWTJobGRDMWxibWRwYm1VNk56QTNNQ0lzSUNKcFlYUWlPakUzTXpFME5qZ3hOalVzSUNKcGMzTWlPaUpvZEhSd09pOHZhR0YwWTJobGRDMXpkR0ZqYXkxaGNHazZPREE0TUNJc0lDSnpaWEoyWlhKZmRYSnNJam9pYUhSMGNEb3ZMMmhoZEdOb1pYUXRjM1JoWTJzdFlYQnBPamd3T0RBaUxDQWljM1ZpSWpvaU56QTNaREE0TlRVdE9EQmhZaTAwWlRGbUxXRXhOVFl0WmpGak5EVTBObU5pWmpVeUlpd2dJblJ2YTJWdVgybGtJam9pTURGa01XWm1NMkV0Tm1Wak1pMDBOakk0TFRneVlUTXROelUxWXpObE9UZGxaalE1SW4wLmZOazdjck9EalV5Z0N2M3pja2VjVkdmZHIxTXJaRUx2YW1GYnpvQ3kxOEpYVURZR2hXa3BpRlJkMGp5WVF6UlpRdnl4NE54bk9XZ2wyZFJzQ2lrWXlR
+
+# OpenAI
+#  OPENAI_API_KEY:
+#  OPENAI_API_BASE:
+
+#  Anthropic
+#  ANTHROPIC_API_KEY:
+
+# Azure
+  AZURE_API_KEY: Mzk3MmViNGU1MDQwNDM5NmIyM2RhN2M2NTk2YWQxZGU=
+  AZURE_API_BASE: aHR0cHM6Ly9yYWphbi1tMm5xMnB5aS1jYW5hZGFlYXN0Lm9wZW5haS5henVyZS5jb20=
```

Comment:
This should be endpoint if I'm understanding correctly?

Reply:
Yes it is. But it is base63 encoded as per kubernetes secrets requirement.